sdk/iothub/azure-iot-deviceprovisioningservice/README.md

```diff
@@ -0,0 +1,215 @@
+# Azure IoT Hub Device Provisioning Service client library for Python
+
+The IoT Hub Device Provisioning Service (DPS) is a helper service for IoT Hub that enables zero-touch, just-in-time provisioning to the right IoT hub without requiring human intervention, allowing customers to provision millions of devices in a secure and scalable manner. 
```

Comment:
tiny nit,  "...just-in-time provisioning to the right IoT **H**ub without require human ...."